dist/index.d.ts

```diff
@@ -17,18 +17,45 @@ declare module "shp-write" {
   }
 
   export interface DownloadOptions {
-    folder: string;
+    folder?: string;
     filename?: string;
```

Comment:
I could probably make a strong case that this `filename` property should now be in the `ZipOptions` interface, but ironically it is only used in the `download` function.

Reply:
Good point. Change as you see fit, but please update the readme as well!